data/presets/man_made/gantry.json

```diff
@@ -0,0 +1,26 @@
+{
+    "icon": "temaki-toll_gantry",
+    "fields": [
+        "layer"
+    ],
+    "moreFields": [
+        "material",
+        "maxheight",
+        "operator",
+        "ref"
+    ],
+    "geometry": [
+        "line"
+    ],
+    "tags": {
+        "man_made": "gantry"
+    },
+    "terms": [
+        "gantry",
+        "highway gantry",
+        "sign holder",
+        "road sign holder",
+        "sign structure"
```

Comment:
“Mast arm” refers to just the horizontal part, but this is what people often think of these structures as. “Sign assembly” is also relevant.